src/server/game/AI/SmartScripts/SmartScript.cpp

```diff
@@ -4313,20 +4313,25 @@ void SmartScript::OnInitialize(WorldObject* obj, AreaTrigger const* at)
                 go = obj->ToGameObject();
                 LOG_DEBUG("sql.sql", "SmartScript::OnInitialize: source is GameObject {}", go->GetEntry());
                 break;
+            case TYPEID_PLAYER:
+                if (at)
+                {
+                    mScriptType = SMART_SCRIPT_TYPE_AREATRIGGER;
+                    trigger = at;
+                    atPlayer = obj->ToPlayer();
+                    LOG_DEBUG("sql.sql", "SmartScript::OnInitialize: source is AreaTrigger {}, triggered by player {}", trigger->ID, atPlayer->GetGUID().ToString().c_str());
+                }
+                else
+                    LOG_ERROR("sql.sql", "SmartScript::OnInitialize: !WARNING! Player TypeID is only allowed for AreaTriggers");
+                break;
             default:
-                LOG_ERROR("scripts.ai.sai", "SmartScript::OnInitialize: Unhandled TypeID !WARNING!");
+                LOG_ERROR("sql.sql", "SmartScript::OnInitialize: Unhandled TypeID !WARNING!");
```

Comment:
why do we not keep this as scripts.ai.sai?

Reply:
> why do we not keep this as scripts.ai.sai?

We have no conf defaul for the logging while all the others are sql.sql
Best we keep it consistent.